include/swift/AST/DiagnosticsDriver.def

```diff
@@ -175,5 +175,8 @@ WARNING(warn_drv_darwin_sdk_invalid_settings, none,
 
 WARNING(warning_unsupported_driver_option,none,
         "option '%0' is ony supported in swift-driver", (StringRef))
+
+WARNING(old_driver_deprecated,none,
+        "C++ driver is now deprecated; consider avoiding specifying `%0`", (StringRef))
```

Comment:
Maybe "Legacy" instead of, or in-addition to "C++"? I think a Swift user may not necessarily know or care what language the driver is written in, even if they have the new one disabled, so just "C++ driver" coming from Swift may be confusing.

Reply:
Ah, will change to Legacy.